cmab/bandit.py

```diff
@@ -0,0 +1,44 @@
+import statistics
+from typing import Dict, List
+
+import random as rnd
+
+
+class Bandit:
+    # Shorthand for arms to save on dict size:
+    #   S = move to shoot closest enemy, I = camp in base, C = camp close to base
+    __arm_shorts = ('S', 'I', 'C')
```

Comment:
it makes sense? maybe just number it by enumerate?

Reply:
Will do